apparmor/probe.go

```diff
@@ -95,10 +95,10 @@ func (ks *KernelSupport) SupportsFeature(feature string) bool {
 	return ks != nil && ks.features[feature]
 }
 
-// Evaluate checks if the apparmor module is enabled and if all the required features are available.
-func (ks *KernelSupport) Evaluate() (level FeatureLevel, summary string) {
+// SupportLevel checks if the apparmor module is enabled and if all the required features are available.
+func (ks *KernelSupport) SupportLevel() (level SupportLevel, summary string) {
```

Comment:
It feels a little weird that this function is called `SupportLevel()` and it takes an argument of type `SupportLevel`. Should the function be called `GetSupportLevel()`?

Reply:
Yes, this is all overcomplexity that should go away.